CHANGELOG.rst

```diff
@@ -2,6 +2,15 @@
 Changelog
 =========
 
+0.16.0 (YYYY-MM-DD)
+-------------------
+
+* Now requires ``time_machine >= 2.2.0``.
+
+* Add support for having the same item type as a dependency and a result.
```

Comment:
Feel free to suggest some expanded wording.

Reply:
This is the best I could come up with, but still feels kind of confusing :sweat: 

```suggestion
* ``ItemProvider`` now supports page objects that declare a dependency on the 
  same type of item that they return, as long as there is an earlier page 
  object input provider that can provide such dependency.
```